package.json

```diff
@@ -347,6 +347,7 @@
         "watch": "npm run buildScripts && tsc -watch -p ./",
         "postinstall": "node ./node_modules/vscode/bin/install",
         "test": "npm run compile && node ./test-scripts/test.js",
+        "integrationTest": "npm run compile && ts-node ./test-scripts/integrationTest.ts",
```

Comment:
I don't think we need it because we will also need to add end to end tests eventually, and the number of tests and dependencies to run them gets very large.